src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -37,6 +37,8 @@ import type {BrowsingContextStorage} from './browsingContextStorage.js';
 import type {CdpTarget} from './cdpTarget.js';
 
 export class BrowsingContextImpl {
+  static LOGGER_PREFIX = `${LogType.debug}:browsingContext` as const;
```

Comment:
Assuming not adding `readonly` is intentional. Just curious why?

Reply:
Its OK to have it, and i forgot, but  that being said when you have `as const` value it will only work if someone reassigns is the exact same string.

Reply:
![image](https://github.com/GoogleChromeLabs/chromium-bidi/assets/34244704/fca85edc-cb7e-4492-999d-75a3d2cabfe0)